index.css

```diff
@@ -0,0 +1,23 @@
+body{
+    width: 50%;
+    text-align: center;
+    color: black;
+    background-color: gray;
+  }
+  
+  #welcome-section{
+    height: 100vh;
+  }
+  
+  @media screen and(min-width: 480px) {
+    body {
+      width: 100%;
+      text-align: start;
+      background-color: lightgreen;
+    }
+  }
+  
+  #navbar{
+    position: fixed;
+    top: 0;
+  }
```

Comment:
- [ ] In order to place your content in an appealing way, you might consider using  [flexbox](https://github.com/microverseinc/curriculum-html-css/blob/main/flexbox.md) or [grid](https://github.com/microverseinc/curriculum-html-css/blob/main/grid.md) to position the content
- [ ] Also you can consider using classes to style your content, in that order, it can make it less complicated than using id.